beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -92,6 +92,136 @@ func createLightClientBootstrap(ctx context.Context, state state.BeaconState) (*
 	return result, nil
 }
 
+func createLightClientBootstrapCapella(ctx context.Context, state state.BeaconState, block *structs.BlindedBeaconBlockCapella) (*structs.LightClientBootstrapCapella, error) {
+	// assert compute_epoch_at_slot(state.slot) >= ALTAIR_FORK_EPOCH
+	if slots.ToEpoch(state.Slot()) < params.BeaconConfig().CapellaForkEpoch {
+		return nil, fmt.Errorf("light client bootstrap is not supported before Altair, invalid slot %d", state.Slot())
```

Comment:
```suggestion
		return nil, fmt.Errorf("creating Capella light client bootstrap is not supported before Capella, invalid slot %d", state.Slot())
```
Maybe something like this?